Configuration/Skimming/python/PDWG_EXOMONOPOLE_cff.py

```diff
@@ -0,0 +1,65 @@
+import FWCore.ParameterSet.Config as cms
+
+# Unprescale HLT_MET and HLT_SinglePhoton triggers
+import HLTrigger.HLTfilters.hltHighLevel_cfi
+hltMonopole = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone()
+hltMonopole.TriggerResultsTag = cms.InputTag( "TriggerResults", "", "HLT" )
+hltMonopole.HLTPaths = cms.vstring(
+    "HLT_Photon175_v*",
+    "HLT_PFMET300_v*",
+    "HLT_PFMET170_HBHE_BeamHaloCleaned_v*"
+)
+hltMonopole.throw = False
+hltMonopole.andOr = True
+
+# selection of valid vertex
+#primaryVertexFilterForZMM = cms.EDFilter("VertexSelector",
+#    src = cms.InputTag("offlinePrimaryVertices"),
+#    cut = cms.string("!isFake && abs(z) <= 25 && position.Rho <= 2"), 
+#    filter = cms.bool(True),   # otherwise it won't filter the events
+#    )
+
+#Data
+from Configuration.EventContent.EventContent_cff import AODEventContent
+EXOMonopoleAODContent = AODEventContent.clone()
+# Barrel Cluster
+EXOMonopoleAODContent.outputCommands.append('keep *_hybridSuperClusters_*_*')
```

Comment:
hi @srimanob - a few comments for cleanup: 
here, things would be more readable if you used
exoProducts = cms.untracked.vstring('keep ....', 'keep ....', ...)
EXOMonopoleAODContent.outputCommands.extend(exoProducts)

(I'd suggest EXOMonopoleSkimContent instead of AODContent..